pkg/gadgets/trace/tcpconnect/tracer/bpf/tcpconnect.bpf.c

```diff
@@ -214,10 +253,62 @@ exit_tcp_connect(struct pt_regs *ctx, int ret, int ip_ver)
 	}
 
 end:
-	bpf_map_delete_elem(&sockets, &tid);
+	bpf_map_delete_elem(&sockets_per_process, &tid);
 	return 0;
 }
 
+static __always_inline int cleanup_sockets_latency_map(const struct sock *sk)
+{
+	bpf_map_delete_elem(&sockets_latency, &sk);
+	return 0;
+}
+
+static __always_inline int handle_tcp_rcv_state_process(void *ctx, struct sock *sk)
+{
+	struct piddata *piddatap;
+	struct event event = {};
+	u64 ts;
+
+	if (BPF_CORE_READ(sk, __sk_common.skc_state) != TCP_SYN_SENT)
+		return 0;
+
+	piddatap = bpf_map_lookup_elem(&sockets_latency, &sk);
+	if (!piddatap)
+		return 0;
+
+	ts = bpf_ktime_get_ns();
+	if (ts < piddatap->ts)
+		goto cleanup;
+
+	event.latency = ts - piddatap->ts;
+	if (targ_min_latency_ns && event.latency < targ_min_latency_ns)
+		goto cleanup;
+	__builtin_memcpy(&event.task, piddatap->comm,
+			sizeof(event.task));
+	// TODO!
+	//event.tid = piddatap->tid;
```

Comment:
Why is this TODO commented?

Reply:
I wanted also to extend this gadget to provide TID information (https://github.com/inspektor-gadget/inspektor-gadget/issues/1567) but then I realized it was too much and decided to stop. That comment is a leftover, so I removed it.